zerver/tests/test_push_notifications.py

```diff
@@ -1574,9 +1574,20 @@ def test_push_api(self) -> None:
             result = self.client_delete(endpoint, {'token': 'abcd1234'})
             self.assert_json_error(result, 'Token does not exist')
 
-        # Add tokens
-        for endpoint, token in endpoints:
-            # Test that we can push twice
+    def test_push_api_add_and_remove_device_tokens(self) -> None:
+        user = self.example_user('cordelia')
+        self.login_user(user)
+
+        endpoints = [
+            ('/json/users/me/apns_device_token', 'apple-tokenaa'),
+            ('/json/users/me/android_gcm_reg_id', 'android-token-1'),
+            ('/json/users/me/apns_device_token', 'apple-tokenbb'),
+            ('/json/users/me/android_gcm_reg_id', 'android-token-2'),
+        ]
```

Comment:
Can you make this two lists, `no_bouncer_requests` and `bouncer_requests`?  I think that'd be a lot more readable than cutting the list at 2 elements repeatedly.